docs/scenarios/manifests/README.md

```diff
@@ -0,0 +1,65 @@
+## Generating manifests using CPMA and applying them
+
+CPMA analyzes OCP3 configuration files and when possible transforms them to CRs, generated CRs are used to configure OCP4 cluster as `day two` operations.
+
+---
+
+
+### 1. Prerequisites
+
+Prior to working with CPMA you need to deploy an OCP 3.7+ cluster(OCP 3.7, 3.9, 3.10, 3.11 will be supported).
+There are to 2 ways for cpma to obtain desired configurations:
+
+1. SSH, CPMA is capable to connect to cluster using SSH and fetch configurations to a local directory. For using SSH it's required to have a proper private key and SSH user may require sudo permissions for reading configurations.
```

Comment:
I suggest something like: 
`1. Remotely retrieve configuration files using SSH, which are then stored locally (directory determined by '<workdir>' parameter) and under remote hostname sub directory. Proper SSH credentials (user, port and ssh key) are required with user having access to configuration files (sudo access might be necessary).`